Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -511,6 +515,60 @@ public enum WalletManagerMode: Equatable {
     // Equatable: [Swift-generated]
 }
 
+///
+/// The WalletManager's sync depth determines the range that a sync is performed on.
+///
+/// - fromLastConfirmedSend: Sync from the block height of the last confirmed send transaction.
+///
+/// - fromLastTrustedBlock: Sync from the block height of the last trusted block; this is
+///      dependent on the blockchain and mode as to how it determines trust.
+///
+/// - fromCreation: Sync from the block height of the point in time when the account was created.
+///
+public enum WalletManagerSyncDepth: Equatable {
```

Comment:
So the IOS/Android logic is to 'increase the depth' based on User input (within a 24 hour period).  That is, start shallow, go deeper until the issue is resolved.  This interface does not have an implied ordering.  Perhaps something like:

```
var deeper: WalletManagerSyncDepth? { ... }    // stretching vocabulary
var shallower: WalletManagerSyncDept? { ... }
```

Reply:
The ordering seems a bit goofy, as nothing is to say that last confirmed send will be before or after the last checkpoint. Regardless, added.